docs/api/contrib/index.rst

```diff
@@ -11,3 +11,4 @@ The ``yellowbrick.contrib`` package contains a variety of extra tools and experi
    boundaries
    statsmodels
    scatter
+   missing/index
```

Comment:
Thanks for adding documentation for the visualizers; this is a very complete submission, with two very cool visualizers!